pkg/k8shandler/collection.go

```diff
@@ -56,7 +56,7 @@ func CreateOrUpdateCollection(cluster *logging.ClusterLogging) (err error) {
 		retryErr := retry.RetryOnConflict(retry.DefaultRetry, func() error {
 			if exists, cluster := utils.DoesClusterLoggingExist(cluster); exists {
 				if !reflect.DeepEqual(fluentdStatus, cluster.Status.Collection.LogCollection.FluentdStatus) {
-					logrus.Infof("Updating status of Fluentd")
+					logrus.Info("Updating status of Fluentd")
```

Comment:
I think it is useful to print this, but not more than once if looping in the `RetryOnConflict`

Reply:
Would doing something like the following make sense then? Its kind of ugly but prevents us from updating when not necessary...
```
if exists, cluster := utils.DoesClusterLoggingExist(cluster); exists {
	if !reflect.DeepEqual(fluentdStatus, cluster.Status.Collection.LogCollection.FluentdStatus) {
		logrus.Info("Updating status of Fluentd")
		retryErr := retry.RetryOnConflict(retry.DefaultRetry, func() error {
			if exists, cluster := utils.DoesClusterLoggingExist(cluster); exists {
				if !reflect.DeepEqual(fluentdStatus, cluster.Status.Collection.LogCollection.FluentdStatus) {
					cluster.Status.Collection.LogCollection.FluentdStatus = fluentdStatus
					return sdk.Update(cluster)
				}
			}
			return nil
		})
		if retryErr != nil {
			return fmt.Errorf("Failed to update Cluster Logging Fluentd status: %v", retryErr)
		}
	}
}
```